### PR TITLE
Header.js & Inbox.js Updated

### PR DESCRIPTION
**Pull request by nickelliston**
Updated header.js and inbox.js with bootstrap.  Used Jumbotron for Header.

## What was done
- Updated Header.js with Jumbotron using Bootstrap
- Formatted Inbox.js using Form
-

### CODE REVIEW

File: website/src/components/Header/Header.js [hunk 1]
```diff
@@ -1,16 +1,28 @@
...
-          This is BottlenoseMail!
-        </h1>
-
-        <Button href="#">Create New Inbox</Button>
+        <Jumbotron>
+          <Container>
+            <h1 className="display-4" style={{ color: "black" }}>
+              Bottlenose Mail
+            </h1>
+            <p className="lead" style={{ color: "black" }}>
```
Comment (frankolson) [inline website/src/components/Header/Header.js:13]: This is more of a style preference, but I like to include spaces in between major HTML elements.
```suggestion
            </h1>
            
            <p className="lead" style={{ color: "black" }}>
```

File: website/src/components/Header/Header.js [hunk 2]
```diff
@@ -1,16 +1,28 @@
...
+              Forget about spam, advertising mailings, hacking and attacking
+              robots. Keep your real mailbox clean and secure. Bootlenoose Mail
+              provides a temporary, secure, anonymous, free, disposable email
+              address.
+            </p>
+            <br />
```
Comment (frankolson) [inline website/src/components/Header/Header.js:19]: We should use margins instead of breaklines.

File: website/src/views/Inbox/Inbox.js [hunk 3]
```diff
@@ -1,14 +1,15 @@
...
 
 export default function Inbox({ match }) {
   return (
     <Row as="main">
       <Col>
-        <p className="text-center">
-          Here is inbox: {match.params.inbox}.
-        </p>
+        <Form.Group controlId="exampleForm.ControlTextarea1" sm>
+          <Form.Label style={{ color: "black" }}>Inbox</Form.Label>
```
Comment (frankolson) [inline website/src/views/Inbox/Inbox.js:9]: We should use css classes instead of inline styles.

**Review by frankolson — CHANGES_REQUESTED** [review]

**Comment by nickelliston** [comment]
Closing in favor of #19

**Pull request closed without merging**